### DISCUSSION TITLE
[Feature] Ability to disable new v1.107 feature: Display all local assets on the mobile app timeline

### DISCUSSION THREAD

**Discussion by RealJohnGalt** [immich-app/immich#10772] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This feature worsens the already poor performance of the mobile app by blocking thumbnail loading (including of immich assets) until local assets load. On device thumbnail generation for large local assets (for example high resolution dng) is incredibly slow.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10761
